README.md

```diff
@@ -7,7 +7,7 @@
 
 # Contributing
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more information.
```

Comment:
Better to redirect to OpenJDK or provide the link to the runtime or jdk download.

Reply:
Also need specific instructions to install JDK in MacOS/Windows/Linux atleast. I used brew